src/clockcache.c

```diff
@@ -2128,7 +2128,25 @@ clockcache_get_internal(clockcache   *cc,       // IN
    platform_status   status;
    uint64            start, elapsed;
 
-   debug_assert(allocator_get_ref(cc->al, base_addr) > 1);
+#if SPLINTER_DEBUG
+   uint8 extent_ref_count = allocator_get_ref(cc->al, base_addr);
+
+   // Dump allocated extents info for deeper debugging.
+   if (extent_ref_count <= 1) {
+      allocator_print_allocated(cc->al);
```

Comment:
This may be more than we want ... but let's merge this and watch how things work out.

The said assertion has not reappeared in last 2-3 some weeks.